.kiro/specs/20260325-knowledge-system/evidence.md

```
@@ -0,0 +1,603 @@
+
+## 1. Scrapboxのリンクベース設計 × Zettelkasten
+
+### 設計判断
+> 知識の蓄積・接続方法として Zettelkasten メソッドを採用し、Scrapboxをその実装基盤とする
+
+### 根拠
+
+#### 1-1. Scrapbox開発者 増井俊之「階層的管理は常人には難しい」
+
+> 個人的な雑多なデータを階層的に管理することは常人にはとても難しい
+
```

markdownlint-cli2 (0.22.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)